protocol.html

```diff
@@ -486,81 +516,41 @@ <h3 property="schema:name">Discovery</h3>
-              </section>
-
-
-              <section id="notification-gateway" inlist="" rel="schema:hasPart" resource="#notification-gateway">
-                <h3 property="schema:name">Notification Gateway</h3>
-                <div datatype="rdf:HTML" property="schema:description">
-                  <p>Once the location of a notification gateway is discovered, an application will negotiate for a particular notification channel, based on acceptable features and protocols.</p>
-
-                  <p>The <code>type</code> array advertises which subscription channels a client is able to understand. The <code>features</code> array advertises which notification features are required by the client. The server selects one of the proposed subscription types and responds with a JSON-LD document that includes the selected protocol, the endpoint for negotiating a subscription and the supported features offered by that protocol.</p>
-
-                  <p>Authentication is not required at this endpoint.</p>
-
-                  <p class="advisement">This definition of a gateway API is <em>at risk</em> and will likely be changed in a future version of this document.</p>
-
-                  <p>A sample interaction is described below. The type and feature names are included as examples.</p>
-
-                  <figure id="notification-gateway-request" class="example listing" rel="schema:hasPart" resource="#notification-gateway-request">
-                    <p class="example-h"><span>Example</span>: Requesting subscription types and features from the Notification Gateway.</p>
-
-                    <pre about="#notification-gateway-request" property="schema:description" typeof="fabio:Script"><code>POST /gateway</code>
-<code>Content-Type: application/ld+json</code>
+<code>@prefix solid: &lt;http://www.w3.org/ns/solid/terms#&gt; .</code>
+<code>@prefix notify: &lt;http://www.w3.org/ns/solid/notifications#&gt; .</code>
 <code></code>
-<code>{</code>
-<code>  "@context": ["https://www.w3.org/ns/solid/notification/v1"],</code>
-<code>  "type": ["WebSocketSubscription2021", "WebHookSubscription2021"],</code>
-<code>  "features": ["state", "rate"]</code>
-<code>}</code></pre>
-
-                    <figcaption property="schema:name"><code>POST</code> request including <code>type</code> and <code>features</code> targeting the Notifications Gateway.</figcaption>
-                  </figure>
-
-                  <figure id="notification-gateway-response" class="example listing" rel="schema:hasPart" resource="#notification-gateway-response">
-                    <p class="example-h"><span>Example</span>: Response from the Notification Gateway including a type of subscription endpoint with available features.</p>
-
-                    <pre about="#notification-gateway-response" property="schema:description" typeof="fabio:Script"><code>HTTP/2</code>
-<code>Content-Type: application/ld+json</code>
+<code>&lt;&gt;</code>
+<code>  a solid:StorageMetadata ;</code>
```

Comment:
Just noting that something like this is what we'll probably end up with but there is a slight mismatch here with the description around it in that the resource is about server-metadata but `<>` is about storage metadata.

Same goes for the Protocol section starting off with "capabilities of a Solid Storage" but the text has emphasis on server.